Configurations/15-android.conf

```diff
@@ -40,20 +20,33 @@
             die "\$ANDROID_NDK is not defined"  if (!$ndk);
             die "\$ANDROID_NDK=$ndk is invalid" if (!-d "$ndk/platforms");
 
+            my $ndkver = undef;
+
+            if (open my $fh, "<$ndk/source.properties") {
+                my $_;
```

Comment:
Maybe you meant `local $_`?
(I have never done that myself, so dunno if it actually works)

Reply:
Just in case for reference. If one doesn't limit $_'s scope here, then last line of $ndk/source.properties ends up in the middle of configdata.pm as additional key in %target. Another thing to note is that I use perl 5.10 for test builds on Linux in order to ensure that we don't exceed the minimum requirement. And `my $_` did work for me. But sure, if it doesn't work with latest perl, then it needs to be addressed. And yes, local is the answer here...